oasst-shared/oasst_shared/schemas/inference.py

```diff
@@ -178,6 +178,19 @@ class Thread(pydantic.BaseModel):
     messages: list[MessageRead]
 
 
+class SafetyParameters(pydantic.BaseModel):
+    enabled: bool = True
```

Comment:
would set to `False` by default bc if this gets merged & released without the safety server being up yet, we want it to be "backwards compatible"